Source/JavaScriptCore/parser/Parser.cpp

```
@@ -3281,6 +3313,9 @@ template <class TreeBuilder> TreeSourceElements Parser<LexerType>::parseClassFie
                 type = DefineFieldNode::Type::ComputedName;
                 break;
             }
+            case OPENBRACE: {
```

What happens if `isStaticField` is false?

---

It should throw a syntax error.